src/main/java/fr/insee/sabianedata/ws/model/pearl/Visibility.java

```diff
@@ -32,6 +33,12 @@ public class Visibility {
     private Long interviewerStartDate;
     private Long managementStartDate;
     private Long endDate;
+    @Getter
+    private final String mail="no-mail-yet";
+    @Getter
+    private final String tel = "no-tel-yet";
+    @Getter
+    private final boolean useLetterCommunication = false;
```

Comment:
J'ai l'impression qu'on est pas loin de pouvoir passer tous les attributs en `final` et convertir la classe en `record` (mais ça demande peut-être encore du taff)

NB : pour les valeurs par défaut avec un record, je suis tombé sur cette question stackoverflow https://stackoverflow.com/questions/73646644/default-values-for-record-properties